net.sourceforge.Artha.yaml

```diff
@@ -0,0 +1,140 @@
+app-id: net.sourceforge.Artha
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: artha
+
+rename-desktop-file: artha.desktop
+#rename-icon: artha.png
+
+finish-args:
+  - --socket=x11
+  - --device=dri
+  # dbus session bus for artha
+  - --own-name=org.artha.unique
+  
+modules:
+
+- shared-modules/gtk2/gtk2.json 
+    
+# Wordnet 3.0 Needs tcl 8.5.x 
+- name: tcl8.5.19
+  cleanup: 
+    - /cache
+    - /man
+    - /share/aclocal
+    - /share/devhelp
+    - /include
+    - /lib/pkgconfig
+    - /share/gtk-doc
+    - /share/man
+    - /lib/*.a
+    - '*.a'
+    - '*.la'
+  sources:
+  - type: archive
+    url: ftp://ftp.tcl.tk/pub/tcl/tcl8_5/tcl8.5.19-src.tar.gz
+    sha256: d3f04456da873d17f02efc30734b0300fb6c3b85028d445fe284b83253a6db18
+  subdir: unix
+  build-options:
+    no-debuginfo: true
+  post-install:
+  - make install-private-headers 
+  - ln -v -sf tclsh8.6 ${FLATPAK_DEST}/bin/tclsh 
+  - chmod -v 755 ${FLATPAK_DEST}/lib/libtcl8.5.so
+
+# Wordnet 3.0 Needs tk 8.5.x
+- name: tk8.5.19
+  cleanup: 
+    - /cache
+    - /man
+    - /share/aclocal
+    - /share/devhelp
+    - /include
+    - /lib/pkgconfig
+    - /share/gtk-doc
+    - /share/man
+    - /lib/*.a
+    - '*.a'
+    - '*.la'
+  sources:
+  - type: archive
+    url: ftp://ftp.tcl.tk/pub/tcl/tcl8_5/tk8.5.19-src.tar.gz
+    sha256: 407af1de167477d598bd6166d84459a3bdccc2fb349360706154e646a9620ffa
+  subdir: unix
+  post-install:
+  - make install-private-headers 
+  - ln -v -sf wish8.6 ${FLATPAK_DEST}/bin/wish 
+  - chmod -v 755 ${FLATPAK_DEST}/lib/libtk8.5.so
+
+- name: wordnet
+  cleanup: 
+    - /cache
+    - /man
+    - /share/aclocal
+    - /share/devhelp
+    - /include
+    - /lib/pkgconfig
+    - /share/gtk-doc
+    - /share/man
+    - /lib/*.a
+    - '*.a'
+    - '*.la'
+  buildsystem: autotools
+  config-opts:
+  - --with-tcl=/app/lib
+  - --with-tk=/app/lib
+  - --enable-static=no 
+  sources:
+  - type: archive
+    url: http://wordnetcode.princeton.edu/3.0/WordNet-3.0.tar.bz2
+    sha256: 6c492d0c7b4a40e7674d088191d3aa11f373bb1da60762e098b8ee2dda96ef22
+    patch: 
+    
+- name: dbus-glib
```

Comment:
this is also in shared-modules.

Reply:
This wasn't solved...

Reply:
made changes as advised https://github.com/flathub/flathub/pull/4670